src/ui/public/doc_viewer/doc_viewer.js

```diff
@@ -48,6 +60,13 @@ uiModules.get('kibana')
     controller: function ($scope) {
       $scope.mode = docViews.inOrder[0].name;
       $scope.docViews = docViews.byName;
+
+      $scope.onKeyUp = function onKeyUp(e, name) {
```

Comment:
I think the directive I made in https://github.com/elastic/kibana/pull/11604 will help me with this. I'll merge that PR first and then try to fix this.